### DISCUSSION TITLE
How can I install and enable dxvk in gamehub for a wine game?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#522] [Q&A]
A wine game doesn't work and that could have two reasons:
1.It doesn't work with the normal Wine,then I would have to wait until support for more wine versions is supported by gamehub.
2.dxvk isn't enabled.
If it's the second thing,how can I do that?

**Comment by Lucki** [comment]
> how can I do that?

Winetricks is able to install that, Proton has it built in.

**Reply by ghost** [reply]
So will it not be installed for every wine game you add automatically?
Cause that would make much easier.
If that isn't the case,it would be cool if the devs implement it in the future.
